src/storage/buxfer.ts

```diff
@@ -117,10 +117,19 @@ export class BuxferStorage implements TransactionStorage {
       accountId: Number(accountId),
       date: format(parseISO(tx.date), BUXFER_DATE_FORMAT, {}),
       amount: tx.chargedAmount,
-      description: tx.description,
-      status:
-        tx.status === TransactionStatuses.Completed ? "cleared" : "pending",
+      description: this.createCompositeDescription(tx),
+      // TODO - uncomment once the API bug is resolved on Buxfer service --> status: = tx.status === TransactionStatuses.Completed ? "cleared" : "pending",
```

Comment:
Can you remove the field in the `buxfer-ts-client` code? Then when they fix it the only change we need in moneyman is the version

```suggestion
      description: this.createCompositeDescription(tx),
      status: tx.status === TransactionStatuses.Completed ? "cleared" : "pending",
```

Reply:
Got it, will do.